http/server.go

```diff
@@ -112,3 +114,42 @@ func GetRequestID(r *http.Request) string {
 func GetUserID(r *http.Request) string {
 	return strx.Or(r.Header.Get("X-User"), r.Header.Get("X-User-Id"))
 }
+
+// GetSkipAndCount returns the skip and count pagination values from the query parameters
+// skip is the number of records to pass over before starting a search
+// count is the number of records to retrieve in the search
+// exists indicates if skip or count was passed into the request URL
+func GetSkipAndCount(r *http.Request) (skip int, count int, exists bool, errors []error) {
+	skipVal := r.URL.Query().Get("skip")
+	var err error
+	skip, err = strconv.Atoi(skipVal)
+	if err != nil && len(skipVal) > 0 {
+		errors = append(errors, err)
+		skip = 0
+	}
+	// TODO - Add this to a config somewhere?
+	// Cap skip at 10000
```

Comment:
I don't think we care about these being configurable. They're going to be hardcoded across services.

Reply:
Makes sense. Pushed.